doc/manual/source/api-endpoints.rst

```diff
@@ -12,7 +12,8 @@ The HTTP service supports GET requests on the following paths:
 ``/api/v1/validity/as-number/prefix``
      Returns a JSON object describing whether the route announcement given by
      its origin AS Number and address prefix is RPKI valid, invalid, or not
-     found. A complete list of VRPs that caused the result is included.
+     found. See :doc:`validity checker<validity-checker>`. A complete list of
```

Comment:
I would flip this around: “A complete list […] is included. For details about its content see ….“